### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TomoLV**
Basic description of example app functionalities

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -2,15 +2,98 @@
...
+
+PeripheralListScreen combines two features:
+- Control of scan process (start / stop), which is represented by a 
+button in the AppBar
+- List of scanned peripherals, which takes the main space on the 
+screen
+
+Scanned peripherals are represented in a list. Each peripheral row 
+contains:
+- Peripheral icon based on peripheral category,
```
Comment (martawoldanska) [inline example/README.md]: Please start each bullet point with small letter.

File: example/README.md [hunk 2]
```diff
@@ -2,15 +2,98 @@
...
-- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
+![](./images/ui-flow.gif)
 
-For help getting started with Flutter, view our
-[online documentation](https://flutter.dev/docs), which offers tutorials,
-samples, guidance on mobile development, and a full API reference.
+### PeripheralListScreen
+
+PeripheralListScreen combines two features:
+- Control of scan process (start / stop), which is represented by a
```
Comment (martawoldanska) [inline example/README.md]: Please start each bullet point with small letter.

File: example/README.md [hunk 3]
```diff
@@ -2,15 +2,98 @@
 
 Demonstrates how to use the blemulator plugin.
 
-## Getting Started
+## UI structure
 
-This project is a starting point for a Flutter application.
+App consists of two screens: PeripheralListScreen and
```
Comment (martawoldanska) [inline example/README.md]: For the names taken straight from the code I'd use grave accents (`).

File: example/README.md [hunk 4]
```diff
@@ -2,15 +2,98 @@
...
+contains:
+- Peripheral icon based on peripheral category,
+- Peripheral identifier,
+- Peripheral name,
+- Rssi value with appropriate icon and color depending on signal 
+strength.
+
+Color of peripheral icon and identifier text is based on category.
+
+Tapping on a given peripheral row results in a transition to the
```
Comment (martawoldanska) [inline example/README.md]: This sentence is unnatural to me.

File: example/README.md [hunk 5]
```diff
@@ -2,15 +2,98 @@
...
+**Tabbed layout with 3 tabs** - for SensorTag peripherals (primary peripheral
+used during development):
+1. Base details view (exactly the same as for all other peripherals)
+2. Automated test view
+3. Manual test view
+
+##### Base details view
+
+Base details view is a scrollable view that contains specific sections,
+responsible
```
Comment (martawoldanska) [inline example/README.md]: `responsible`... for what? ;)

File: example/README.md [hunk 6]
```diff
@@ -2,15 +2,98 @@
...
+
+Each CharacteristicItem contains:
+- characteristic identifier,
+- characteristic value,
+- characteristic properties (isReadableWithResponse, isReadableWithoutResponse,
+isNotifiable, isIndicatable),
+- characteristic operation buttons: Read, Write, Monitor 
+(which are available depending on properties).
+
+Pressing Read, Write, Monitor buttons results in an appropriate actions:
```
Comment (martawoldanska) [inline example/README.md]: Please rephrase to "appropriate action".

**Review by martawoldanska — CHANGES_REQUESTED** [review]

File: example/README.md (hunk 5 above)
Comment (TomoLV) [reply example/README.md]: Looks like a leftover... :). Removed.

File: example/README.md (hunk 4 above)
Comment (TomoLV) [reply example/README.md]: Rephrased both sentences to:
```
Peripheral category determines the color of icon view and identifier
 text view.

 When peripheral row is tapped, application transitions to the 
 `PeripheralDetailsScreen`.
```

**Review by martawoldanska — APPROVED** [review]

File: example/README.md [hunk 7]
```diff
@@ -2,15 +2,98 @@
...
-This project is a starting point for a Flutter application.
+App consists of two screens: `PeripheralListScreen` and 
+`PeripheralDetailsScreen`.
 
-A few resources to get you started if this is your first Flutter project:
+Skeleton UI flow:
 
-- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
-- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
+![](./images/ui-flow.gif)
```
Comment (mikolak) [inline example/README.md:12]: A static map of screens would be more useful and lightweight.

Comment (lukerayman) [reply example/README.md:12]: It has very low fps. And I think it should be smaller. I don't know if you know, but you can use `` tag with `width` attribute to set the size, so the GIF image does not need to be downscaled itself.

**Pull request closed without merging**